.storybook/babel.config.js

```diff
@@ -16,4 +16,35 @@
  * limitations under the License.
  */
 
-module.exports = require('../babel.config.js');
+module.exports = {
```

Comment:
This file now writes the babel config instead of importing it as we need to set the babel targets which we can't do at the root as it causes compliation errors for our outputed css in bpk-stylesheets, so we just modify the babel config to accept the correct targets

Reply:
As a lot of this file is similar, it might be worth importing the root one as before and then changing anything we need to?
However I can see the counter-argument that says these are effectively config files for 2 separate use cases, so duplicating isn't necessarily a bad idea?

Reply:
Might be worth adding a comment to the root one though stating that the storybook config is separate/different

Reply:
Sure will add the appropriate comments! 

I tried to keep them the same and just doing an import and modifying all that we needed but build/running didn't like that.

So the only option was to have a separate config, but as the point you mentioned they are specific configs for different usecases so should be ok.